`Text` component - Documentation (#1550)
hashicorp/design-system: website/docs/components/text/partials/accessibility/accessibility.md

## Conformance rating

<!-- Update conformance rating badge with correct status and remove the others -->
<Doc::Badge @type="success">Conformant</Doc::Badge>

When used as recommended, there should not be any WCAG conformance issues with this component.

!!! Info

> Since this component is nothing more than a wrapper around generic HTML tags with custom typographic styles applied, any conformance issue that may arise will depend on how this component is used in the consumer's code, in the same way as normal HTML tags and text would do.

Reviewer: ```suggestion
Since this component is nothing more than a wrapper around generic HTML tags with custom typographic styles applied, any conformance issues that may arise will depend on how this component is used in the consumer's code, in the same way as normal HTML tags and text would do.
```

> Since this component is nothing more than a wrapper around generic HTML tags with custom typographic styles applied, any conformance issues that may arise will depend on how this component is used in the consumer's code, in the same way as normal HTML tags and text would do.

Reviewer 2: ```suggestion
As this component is nothing more than a wrapper around generic HTML tags with custom typographic styles applied, any conformance issues that may arise will depend on how this component is used in the consumer’s code, in the same way as normal HTML tags and text would.
```
